testdata/goldens/php/xrefmap.yml

```diff
@@ -0,0 +1,3106 @@
+### YamlMime:XRefMap
```

Comment:
This file can be removed for doc-templates goldens.

Reply:
done!